well_known/views.py

```diff
@@ -7,6 +7,6 @@ def serve_text_file(request, file_name):
     try:
         with open(file_path, 'r') as file:
             content = file.read()
-        return HttpResponse(content, content_type='text/plain')
+        return HttpResponse(content, content_type='text/plain; charset=utf-8')
```

Comment:
Django connaît les deux arguments et s’occupe de les agencer.
```suggestion
        return HttpResponse(content, content_type='text/plain', charset='utf-8')
```

Reply:
Apparemment, il s’en occupe mal. J’ai essayé la suggestion est ça ne fonctionne pas.

Reply:
Ok pour toi pour merge en l'etat ?